packages/compass-e2e-tests/helpers/commands/wait-for-animations.ts

```diff
@@ -13,25 +13,31 @@ export async function waitForAnimations(
     return typeof selector === 'string' ? await browser.$(selector) : selector;
   }
 
-  const initialElement = await getElement();
+  try {
+    const initialElement = await getElement();
```

Comment:
We could try and change the signature to take only `selector` or `ChainablePromiseElement<WebdriverIO.Element>` (and not also `WebdriverIO.Element)` and then change `getElement()` to not `await` so that `initialElement` here and `currentElement` below will be a `ChainablePromiseElement<WebdriverIO.Element>` and not an `Element`, but  then I have to hunt down all the places that uses this and that might pass a `WebdriverIO.Element` to this function...

The reasoning behind getting the element first was to try and get a consistent getLocation() and getSize() since that's two calls and we're trying to get a consistent result. But this clearly doesn't help anyway because the element reference could become stale in between the calls. So if we were to change initialElement to be a ChainablePromiseElement then the two calls could be operating on separate actual DOM elements which opens up the potential for different kinds of subtle bugs.

Reply:
Nevermind - I ended up making the change anyway.